cli/bin/grain.js

```diff
@@ -113,6 +113,7 @@ program
   .graincOption("--lsp", "generate lsp errors and warnings only")
   .graincOption("--no-color", "disable colored output")
   .graincOption("--no-gc", "turn off reference counting garbage collection")
+  .graincOption("--no-bulk-memory", "turn off bulk memory operations")
```

Comment:
What do you think of this description?
```suggestion
  .graincOption("--no-bulk-memory", "polyfill WebAssembly bulk memory instructions")
```

Reply:
I like it!